development/scripts/run-dev-server.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
is there a reason why this doesn't use `/usr/bin/env bash` like the other scripts?